### ISSUE TITLE
Firefox and Internet Explorer on AppVeyor

### ISSUE THREAD

**Issue by Ks89** [Ks89/My-MEAN-website-client#36]

**Comment by Ks89** [comment]
Some erorrs with IE
```
  × should NOT display the project detail, because project._id is wrong
30 12 2016 23:08:11.047:WARN [web-server]: 404: /assets/images/projects/project_icons/byamanager.png
30 12 2016 23:08:11.051:WARN [web-server]: 404: /assets/images/projects/project_icons/spf.png
30 12 2016 23:08:11.054:WARN [web-server]: 404: /assets/images/projects/project_icons/superapp.png
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 170 of 187 SUCCESS (0 secs / 44.486 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 171 of 187 SUCCESS (0 secs / 44.626 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 172 of 187 SUCCESS (0 secs / 44.761 secs)
  ProjectListComponent
    √ can instantiate it
    ---YES---
      √ can display the projectlist page
      √ should check if projects are displayed correctly
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 173 of 187 SUCCESS (0 secs / 44.875 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 174 of 187 SUCCESS (0 secs / 45.001 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 175 of 187 SUCCESS (0 secs / 45.118 secs)
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling register...'
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 175 of 187 SUCCESS (0 secs / 45.118 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling register...'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling register...'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling register...'
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Response'
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 175 of 187 SUCCESS (0 secs / 45.118 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Response'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Response'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Response'
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) LOG: Object{message: 'User with email fake@fake.it registered.'}
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 175 of 187 SUCCESS (0 secs / 45.118 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: Object{message: 'User with email fake@fake.it registered.'}
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: Object{message: 'User with email fake@fake.it registered.'}
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: Object{message: 'User with email fake@fake.it registered.'}
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Done'
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 175 of 187 SUCCESS (0 secs / 45.118 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Done'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Done'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Done'
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 176 of 187 SUCCESS (0 secs / 45.238 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 177 of 187 SUCCESS (0 secs / 45.337 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 178 of 187 SUCCESS (0 secs / 45.571 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 179 of 187 SUCCESS (0 secs / 45.769 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 180 of 187 SUCCESS (0 secs / 45.994 secs)
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling register...'
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 180 of 187 SUCCESS (0 secs / 45.994 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling register...'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling register...'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling register...'
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) ERROR: Object{_body: '{"message":"User already exists. Try to login."}'}
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 180 of 187 SUCCESS (0 secs / 45.994 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) ERROR: Object{_body: '{"message":"User already exists. Try to login."}'}
Firefox 49.0.0 (Windows 8.1 0.0.0) ERROR: Object{_body: '{"message":"User already exists. Try to login."}'}
Firefox 49.0.0 (Windows 8.1 0.0.0) ERROR: Object{_body: '{"message":"User already exists. Try to login."}'}
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 181 of 187 SUCCESS (0 secs / 46.167 secs)
  RegisterComponent
    √ can instantiate it
    ---YES---
      √ can display the register page
      √ can get RouterLinks from template
      √ should register a new local account and redirect to login
      √ should return to login page
    ---ERROR---
      √ should NOT register local account, because all fields are mandatory
      √ should NOT register local account, because passwords must the equal
      √ should NOT register local account, because emails must the equal
      √ should NOT register local account, because you can register only one account with that email
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 182 of 187 SUCCESS (0 secs / 46.245 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 183 of 187 SUCCESS (0 secs / 46.34 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 184 of 187 SUCCESS (0 secs / 46.674 secs)
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling reset...'
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 184 of 187 SUCCESS (0 secs / 46.674 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling reset...'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling reset...'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling reset...'
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Response'
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 184 of 187 SUCCESS (0 secs / 46.674 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Response'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Response'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Response'
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) LOG: Object{message: 'An e-mail has been sent to fake@fake.it with further instructions.'}
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 184 of 187 SUCCESS (0 secs / 46.674 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: Object{message: 'An e-mail has been sent to fake@fake.it with further instructions.'}
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: Object{message: 'An e-mail has been sent to fake@fake.it with further instructions.'}
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: Object{message: 'An e-mail has been sent to fake@fake.it with further instructions.'}
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Done'
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 184 of 187 SUCCESS (0 secs / 46.674 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Done'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Done'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Done'
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 185 of 187 SUCCESS (0 secs / 46.882 secs)
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 186 of 187 SUCCESS (0 secs / 47.051 secs)
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling reset...'
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 186 of 187 SUCCESS (0 secs / 47.051 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling reset...'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling reset...'
Firefox 49.0.0 (Windows 8.1 0.0.0) LOG: 'Calling reset...'
[1A[2K[1A[2K[1A[2KFirefox 49.0.0 (Windows 8.1 0.0.0) ERROR: Object{_body: '{"message":"No account with that token exists."}'}
IE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 186 of 187 SUCCESS (0 secs / 47.051 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0) ERROR: Object{_body: '{"message":"No account with that token exists."}'}
Firefox 49.0.0 (Windows 8.1 0.0.0) ERROR: Object{_body: '{"message":"No account with that token exists."}'}
Firefox 49.0.0 (Windows 8.1 0.0.0) ERROR: Object{_body: '{"message":"No account with that token exists."}'}
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (0 secs / 47.527 secs)
  ResetComponent
    √ can instantiate it
    ---YES---
      √ can display the password reset page
      √ can get RouterLinks from template
      √ should reset local account's password
      √ should return to login page
    ---ERROR---
      √ should NOT reset local account's password
[1A[2K[1A[2K[1A[2KIE 11.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 (1 FAILED) (35.262 secs / 34.85 secs)
Chrome 53.0.2785 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (38.48 secs / 38.164 secs)
Firefox 49.0.0 (Windows 8.1 0.0.0): Executed 187 of 187 SUCCESS (48.036 secs / 47.527 secs)
TOTAL: 1 FAILED, 560 SUCCESS
Finished in 2 mins 1.778 secs / 2 mins 0.541 secs
SUMMARY:
√ 560 tests completed
× 1 test failed
FAILED TESTS:
  ProjectDetailComponent
    ---ERROR---
      × should NOT display the project detail, because project._id is wrong
        IE 11.0.0 (Windows 8.1 0.0.0)
      Expected 'null' to be ''.
         at Anonymous function (webpack:///src/app/pages/project-detail/project-detail.spec.ts:135:6 <- config/karma-test-runner.js:120369:13)
         at ZoneDelegate.prototype.invoke (webpack:///~/zone.js/dist/zone.js:240:0 <- config/karma-test-runner.js:97919:13)
         at ProxyZoneSpec.prototype.onInvoke (webpack:///~/zone.js/dist/proxy.js:79:0 <- config/karma-test-runner.js:97575:13)
         at ZoneDelegate.prototype.invoke (webpack:///~/zone.js/dist/zone.js:240:0 <- config/karma-test-runner.js:97919:13)
         at Zone.prototype.run (webpack:///~/zone.js/dist/zone.js:113:0 <- config/karma-test-runner.js:97792:17)
         at Anonymous function (webpack:///~/zone.js/dist/jasmine-patch.js:102:0 <- config/karma-test-runner.js:97290:13)
         at Anonymous function (webpack:///~/@angular/core/bundles/core-testing.umd.js:91:0 <- config/karma-test-runner.js:3577:21)
         at ZoneDelegate.prototype.invoke (webpack:///~/zone.js/dist/zone.js:240:0 <- config/karma-test-runner.js:97919:13)
         at AsyncTestZoneSpec.prototype.onInvoke (webpack:///~/zone.js/dist/async-test.js:49:0 <- config/karma-test-runner.js:96880:13)
         at ProxyZoneSpec.prototype.onInvoke (webpack:///~/zone.js/dist/proxy.js:76:0 <- config/karma-test-runner.js:97572:13)
         at ZoneDelegate.prototype.invoke (webpack:///~/zone.js/dist/zone.js:240:0 <- config/karma-test-runner.js:97919:13)
         at Zone.prototype.run (webpack:///~/zone.js/dist/zone.js:113:0 <- config/karma-test-runner.js:97792:17)
         at Anonymous function (webpack:///~/@angular/core/bundles/core-testing.umd.js:86:0 <- config/karma-test-runner.js:3572:17)
         at Anonymous function (webpack:///~/zone.js/dist/async-test.js:38:0 <- config/karma-test-runner.js:96869:25)
         at ZoneDelegate.prototype.invokeTask (webpack:///~/zone.js/dist/zone.js:273:0 <- config/karma-test-runner.js:97952:17)
         at Zone.prototype.runTask (webpack:///~/zone.js/dist/zone.js:151:0 <- config/karma-test-runner.js:97830:21)
         at invoke (webpack:///~/zone.js/dist/zone.js:345:0 <- config/karma-test-runner.js:98024:21)
         at data.args[0] (webpack:///~/zone.js/dist/zone.js:1224:0 <- config/karma-test-runner.js:98903:13)
TOTAL: 1 FAILED, 560 SUCCESS
TOTAL: 1 FAILED, 560 SUCCESS
```

**Comment by Ks89** [comment]
Temporary removed projectDetail ERROR 'should NOT display the project detail, because project._id is wrong' test. It's not important at the moment, I can live without this.

I'll fix this error as part of issue #31 using IE on a local Windows VM.
